sdk/core/azure-common/azure/common/credentials.py

```diff
@@ -28,7 +34,35 @@ def get_cli_profile():
     ACCOUNT.load(os.path.join(azure_folder, 'azureProfile.json'))
     return Profile(storage=ACCOUNT)
 
-def get_azure_cli_credentials(resource=None, with_tenant=False):
+
+class _CliCredentials(object):
+    """A wrapper of CLI credentials type that implements the azure-core credential protocol.
+    """
+
+    _DEFAULT_PREFIX = "/.default"
+
+    def __init__(self, cli_profile):
+        if AccessToken is None:  # import failed
+            raise ImportError("You need to install 'azure-core' to use this feature")
+        self._profile = cli_profile
+
+    def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
+
+        if len(scopes) != 1:
+            raise ValueError("Cannot deal with multiple scope: {}".format(scopes))
```

Comment:
Maybe rephrase to something like:

"Multiple scopes are not supported: {}"

?

Reply:
Fixed